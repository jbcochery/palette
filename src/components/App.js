import React from "react";

export default function App() {
  return (
    <div>
      <h1>Palette</h1>
    </div>
  );
}
