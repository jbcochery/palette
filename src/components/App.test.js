import React from "react";
import { render } from "@testing-library/react";

import App from "./App";

it("renders a title", () => {
  const wrapper = render(<App />);
  expect(wrapper.container).toMatchSnapshot();
});
