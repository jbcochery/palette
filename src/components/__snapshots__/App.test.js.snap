// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a title 1`] = `
<div>
  <div>
    <h1>
      Palette
    </h1>
  </div>
</div>
`;
